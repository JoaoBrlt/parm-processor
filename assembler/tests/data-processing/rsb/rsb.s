// Reverse a positive value.
// 0 - 1 = -1
MOVS r0, #1
RSBS r1, r0, #0

// Reverse a positive value.
// 0 - 149 = -149
MOVS r2, #149
RSBS r3, r2, #0

// Reverse a negative value.
// 0 - (-200) = 200
MOVS r4, #200
RSBS r5, r4, #0
RSBS r6, r5, #0

// Reverse same register.
// 0 - 87 = -87
MOVS r7, #87
RSBS r7, r7, #0
