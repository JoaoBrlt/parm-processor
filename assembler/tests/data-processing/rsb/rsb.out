v2.0 raw
2001 4241 2295 4253 24c8 4265 426e 2757 427f defd
